v1/pipeline/steps/cd/gitops-get-app-details-step-info.yaml

```diff
@@ -0,0 +1,32 @@
+title: GitOpsGetAppDetailsStepInfo
+allOf:
+  - $ref: ../../common/step-spec-type.yaml
+  - type: object
+    required:
+    - agentId
+    - applicationName
+    properties:
+      agentId:
+        - type: string
+      applicationName:
+        - type: string
+      hardRefresh:
+        oneOf:
+          - type: boolean
+          - type: string
+$schema: http://json-schema.org/draft-07/schema#
+type: object
+required:
+  - agentId
+  - applicationName
+properties:
+  agentId:
+    - type: string
+  applicationName:
+    - type: string
+  hardRefresh:
+    oneOf:
+      - type: boolean
+      - type: string
```

Comment:
ideally for booleans we should use instead of just string:
```
    - type: string
      pattern: (<\+.+>.*)
      minLength: 1
````
to support runtime inputs and expressions